<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>xtbenc</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="3" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Options</string>
       </attribute>
       <widget class="QWidget" name="formLayoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>291</width>
          <height>171</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Video</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="codecLabel">
           <property name="text">
            <string>Codec</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="ComboBox_VCodec"/>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="ComboBox_GoP"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="goPLabel">
           <property name="text">
            <string>GoP</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="ComboBox_ICQ">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="iCQLabel">
           <property name="text">
            <string>ICQ</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="bitrateLabel_2">
           <property name="text">
            <string>Bitrate</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="ComboBox_VBitrate">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="formLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>20</y>
          <width>301</width>
          <height>171</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="1" column="0">
          <widget class="QLabel" name="codecLabel_2">
           <property name="text">
            <string>Codec</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="ComboBox_ACodec"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="bitrateLabel">
           <property name="text">
            <string>Bitrate</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="ComboBox_ABitrate">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Audio</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Output</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTextBrowser" name="textBrowser"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="pushButton_Destination">
        <property name="text">
         <string>Destination</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Destination">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QRadioButton" name="radioButton_QSV">
        <property name="text">
         <string>QSV</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_X264">
        <property name="text">
         <string>X264</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_HEVC">
        <property name="text">
         <string>HEVC</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_VAAPI">
        <property name="text">
         <string>VAAPI</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton_Source">
        <property name="text">
         <string>Source</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Source">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_Convert">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>Convert</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Convert</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>executeCommand()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>757</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>798</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Destination</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setDestination()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>44</x>
     <y>569</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_HEVC</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setHEVC()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_QSV</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setQSV()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>53</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Source</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setSource()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>43</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_VAAPI</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setVAAPI()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>636</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>634</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_X264</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setX264()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>executeCommand()</slot>
  <slot>setSource()</slot>
  <slot>setDestination()</slot>
  <slot>setQSV()</slot>
  <slot>setX264()</slot>
  <slot>setHEVC()</slot>
  <slot>setVAAPI()</slot>
 </slots>
</ui>
